### PR TITLE
Fix the activation of Google Analytics

### PR DESCRIPTION
**Pull request by HiromuHota**
# TL;DR
This PR fixes the activation of Google Analytics using string 'false' instead of boolean false as the default value of `ENABLE_GA` to align with others
Note that this PR renames `gaDisable` to `gaActive` for consistency.

## Type
 - [x] Bug Fix

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Based on the root cause described in https://github.com/flyteorg/flyteconsole/issues/614, there is a discrepancy in data type between the environment variable, the local variable, and where it is used. This PR fixes the data type at the local variable definition to align with the other two.

## Tracking Issue
fixes https://github.com/flyteorg/flyteconsole/issues/614

## Follow-up issue
NA

### CODE REVIEW

**Comment by HiromuHota** [comment]
Smoke tests
`ENABLE_GA='true' GA_TRACKING_ID='G-0QW4DJWJ20' yarn start` -> Active
`ENABLE_GA='true' yarn start` -> Inactive
`GA_TRACKING_ID='G-0QW4DJWJ20' yarn start` -> Inactive
`yarn start` -> Inactive

**Comment by HiromuHota** [comment]
@EngHabu I'd appreciate your review!

**Review by EngHabu — APPROVED** [review]
LGTM @jsonporter can you take a look?

**Comment by HiromuHota** [comment]
Did the same smoke tests as of [c375532](https://github.com/flyteorg/flyteconsole/pull/652/commits/c375532e82522ccc79136ab7003b40f86196f226).
@jsonporter I'd appreciate your review.

**Review by jsonporter — APPROVED** [review]
